### PR TITLE
[GUI] Add address-label to sendconfirm-popup

### PR DESCRIPTION
**Pull request by random-zebra**
Replaces #1311

### CODE REVIEW

**Comment by random-zebra** [comment]
Rebased.

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 1]
```diff
@@ -335,36 +342,83 @@ background:transparent;
...
+                         <widget class="QLabel" name="textSendLabel">
+                          <property name="text">
+                           <string>Name/Label</string>
+                          </property>
+                         </widget>
+                        </item>
+                        <item>
+                         <widget class="QLabel" name="textSend">
+                          <property name="text">
+                           <string>D7VFR83SQbiezrW72hjcWJtcfip5krte2Z</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:414]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 2]
```diff
@@ -263,29 +263,36 @@ background:transparent;
...
+                       </property>
+                       <property name="text">
+                        <string>ID</string>
+                       </property>
+                      </widget>
+                     </item>
+                     <item>
+                      <widget class="QLabel" name="textId">
+                       <property name="text">
+                        <string>492526e7fa3c810b35016...40a5df85ee227ab00b1156994</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:291]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 3]
```diff
@@ -444,12 +498,31 @@ background:transparent;
...
+                   <property name="bottomMargin">
+                    <number>9</number>
+                   </property>
+                   <property name="horizontalSpacing">
+                    <number>12</number>
+                   </property>
+                   <item row="1" column="0">
+                    <widget class="QLabel" name="textAmount">
+                     <property name="text">
+                      <string>2 PIV</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:521]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 4]
```diff
@@ -512,35 +550,48 @@ background:transparent;
...
-                     <property name="text">
-                      <string/>
-                     </property>
-                    </widget>
+                   <item row="1" column="3">
+                    <layout class="QHBoxLayout" name="horizontalLayout">
+                     <item>
+                      <widget class="QLabel" name="textInputs">
+                       <property name="text">
+                        <string>1 Inputs</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:565]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 5]
```diff
@@ -603,36 +672,88 @@ background:transparent;
...
+                      </property>
+                      <property name="text">
+                       <string>Fee</string>
+                      </property>
+                     </widget>
+                    </item>
+                    <item>
+                     <widget class="QLabel" name="textFee">
+                      <property name="text">
+                       <string>0.0001 PIV</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:706]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 6]
```diff
@@ -603,36 +672,88 @@ background:transparent;
...
+                       </property>
+                       <property name="alignment">
+                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+                       </property>
+                      </widget>
+                     </item>
+                     <item>
+                      <widget class="QLabel" name="textSize">
+                       <property name="text">
+                        <string>2 kB</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:743]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 7]
```diff
@@ -707,88 +850,40 @@ background:transparent;
...
                      </property>
                     </widget>
                    </item>
-                   <item>
-                    <widget class="QLabel" name="textSize">
+                   <item row="1" column="1">
+                    <widget class="QLabel" name="textConfirmations">
                      <property name="text">
-                      <string>2 kB</string>
+                      <string>12</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:879]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui [hunk 8]
```diff
@@ -335,36 +342,83 @@ background:transparent;
...
+                        <property name="rightMargin">
+                         <number>0</number>
+                        </property>
+                        <property name="bottomMargin">
+                         <number>0</number>
+                        </property>
+                        <item>
+                         <widget class="QLabel" name="textSendLabel">
+                          <property name="text">
+                           <string>Name/Label</string>
```
Comment (Fuzzbawls) [inline src/qt/pivx/forms/sendconfirmdialog.ui:407]: Might as well just use a dummy non-translated string here like
```xml
<string notr=true>N/A</string>
```

File: src/qt/pivx/forms/sendconfirmdialog.ui (hunk 1 above)
Comment (random-zebra) [reply src/qt/pivx/forms/sendconfirmdialog.ui:414]: `notr="true"` rather than `notr=true`, but yeah.  :+1:

**Comment by random-zebra** [comment]
Fixed dummy non-translated strings in `sendconfirmdialog.ui`.

**Pull request merged**